### DISCUSSION TITLE
Any forks of this card in active development?

### DISCUSSION THREAD

**Discussion by deltamelter** [RomRider/apexcharts-card#633] [Q&A]
Apex charts card is great and I've been able to get a lot out of it... but using it more is killing the browser, most probably due to the 100k+ console messages eg
`Error: <path> attribute d: Expected number, "M 0 NaN"`.
https://github.com/RomRider/apexcharts-card/issues/17

@RomRider Do  you still have time to maintain/develop this? Need help? Are there any active forks or developers looking to perhaps pick up the baton and keep this project alive?

**Comment by metz-spg** [comment]
Having the same question. Apexcharts is still recommended in forums etc. but as it seems to be no longer actively maintained I wonder if I should still use it for my Home-Assistant dashboards.

**Comment by wormuths** [comment]
This is really unfortunate.  It seems that this is a dead-end, because no matter how many bugs get filed, they just sit there.  I love the customization of the card, but some basic functionality hasn't been added, like "difference" for statistics to compare month over month.  

Maybe we can start a bounty-type thing on Home Assistant forums to chip in for someone to take over?  If we all throw $5, $10, $20 bucks into a go-fund-me or something... maybe a small amount of incentive can get someone to step up and spend some time tinkering with the pull requests and smaller bugs?

**Comment by paradox460** [comment]
For things ApexCharts can't do, I've been using Plotly: https://github.com/dbuezas/lovelace-plotly-graph-card

Its a lot more involved to get something that looks as good as ApexCharts, but you have a _lot_ more levers to pull too.

I've used it to make a wind rose card:
<img width="390" alt="CleanShot 2024-03-09 at 20 48 38@2x" src="https://github.com/RomRider/apexcharts-card/assets/168193/7c8d8b1d-921d-4820-9c1b-a86c927d5764">

And a somewhat simple bar graph of driving costs, which uses a lot of calculations behind the scenes to get its data (namely, that the prices of gas, the distance driven, and the amount of gas used as reported by the car)
<img width="499" alt="CleanShot 2024-03-09 at 20 48 51@2x" src="https://github.com/RomRider/apexcharts-card/assets/168193/2cd7731c-4cc1-442b-8cc7-e133e717c687">

**Reply by deltamelter** [reply]
I've also been playing with plotly card for a few days now...  slowly making is do all the things Apex can do and much more on top
![image](https://github.com/RomRider/apexcharts-card/assets/27061628/dafa1f5e-c000-4fff-9409-131a35a53d00)
The main thing that you notice straight away is the performance and interaction speed.  
Some of my Apex graphs are unusable now because of the javascript errors, they take 5 mins to display...
